src/components/Questionnaire/QuestionnaireForm.tsx

```diff
@@ -98,6 +98,20 @@ export function QuestionnaireForm({
     }
   }, [questionnaireData, isInitialized, questionnaireSlug]);
 
+  useEffect(() => {
+    if (questionnaireForms.length > 0) {
+      setTimeout(() => {
+        const lastElement = document.getElementById(
+          `${questionnaireForms[questionnaireForms.length - 1].questionnaire.id}-${questionnaireForms.length - 1}`,
+        );
+
+        if (lastElement) {
+          lastElement.scrollIntoView({ behavior: "smooth", block: "start" });
+        }
+      }, 200);
+    }
+  }, [questionnaireForms]);
```

Comment:
lets try someother approach 🤔

Reply:
> lets try someother approach 🤔

Hmm, how about keeping the questionnaire selector component as sticky then? 🤔